data/api/client-server/relations.yaml

```
@@ -141,7 +389,8 @@ paths:
                 title: "ChildEventsChunk"
                 type: array
                 description: |-
-                  The child events which point to the parent event, after applicable filters.
+                  The child events of the requested event. If a `relType` or `eventType` was
+                  supplied on the URL, the events returned will match those details.
```

```suggestion
                  The child events of the requested event. The events returned will 
                  match the `relType` or `eventType` supplied in the URL.
```

Likewise above (with modifications) x2